test/async-hooks/init-hooks.js

```diff
@@ -147,7 +152,7 @@ class ActivityCollector {
       // events this makes sense for a few tests in which we enable some hooks
```

Comment:
nit: If you're already here, could you turn this comment to a proper paragraph (capitalization, some punctuation, etc)